docs/developer/index.rst

```diff
@@ -41,7 +41,47 @@ Sometimes a PR is still in development and known to fail the tests or
 simply does not touch any code, because it only modifies docstrings
 and the documentation. In that case, `[skip ci]` can be added to the
 commit message to prevent running the github actions tests to save
-time, energy, and limited free recources.
+time, energy, and limited recources.
+
+Run tests locally
+-----------------
+Before you issue a pull request, we ask to run the test suite locally.
+Assuming everything is set up to install Sherpa from source, it can be
+installed in development mode with ``pip``::
+
+  pip install -e .
+
+"Development mode" means that the tests will pick up changes in the
+Python source files without running ``pip`` again (which can take some
+time). Only if you change the C code, you will have to explicitly run
```

Comment:
Technically for us it's C++ and not C code, but I'm not sure how pedantic we should be here.